modules/logging-loki-reliability-hardening.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: REFERENCE
+[id="logging-loki-reliability-hardening_{context}"]
+= Configuring Loki to tolerate node failure
+
+In logging 5.8 and newer versions, the Loki Operator supports setting pod anti-affinity rules to request that pods of the same component are scheduled on different available nodes in the cluster.
+
+include::snippets/about-pod-affinity.adoc[]
+
+[id="logging-loki-reliability-hardening-default-settings_{context}"]
+== Default Loki settings
+
+The Operator sets default, preferred `podAntiAffinity` rules for all Loki components, which includes the `compactor`, `distributor`, `gateway`, `indexGateway`, `ingester`, `querier`, `queryFrontend`, and `ruler` components.
+
+.Example default preferred rule settings for the ingester component
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: lokistack-dev
+  namespace: openshift-logging
+spec:
+# ...
+  template:
+    ingester: <1>
+      podAntiAffinity: <2>
+        preferredDuringSchedulingIgnoredDuringExecution: <3>
+        - podAffinityTerm:
+            labelSelector:
+              matchLabels: <4>
+                app.kubernetes.io/component: ingester <5>
+                app.kubernetes.io/instance: lokistack-dev <6>
+            topologyKey: kubernetes.io/hostname <7>
+          weight: 100
+# ...
+----
+<1> The component `template` spec. This value can be `compactor`, `distributor`, `gateway`, `indexGateway`, `ingester`, `querier`, `queryFrontend`, or `ruler`, depending on the component being configured. Multiple components are included in one `LokiStack` custom resource (CR).
+<2> The stanza to configure node anti-affinity.
+<3> The stanza to define a preferred rule.
+<4> The key/value pair (label) that must be matched to apply the rule.
+<5> The label for the component being configured. This value matches the field name from the component `template` spec `<1>`.
+<6> The namespace where the Loki components are deployed.
```

Comment:
Here it's actually the name of the LokiStack CR that  creates the different resources, not the namespace where they are deployed
```suggestion
<6> The name of the LokiStack instance.
```

Reply:
Ended up removing this default example, since QE pointed out it wasn't clear that this didn't need to be configured